src/components/DonationTrackingTable/DonationSite.jsx

```diff
@@ -1,23 +1,50 @@
 /* eslint react/prop-types: 0 */
 import { Tr, Td, Checkbox } from '@chakra-ui/react';
+import { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useBackend } from '../../contexts/BackendContext';
 
 const DonationSite = ({ donation_site, checkSet, setCheck, topCheckBox }) => {
+  const [individualCheckBox, setIndividualCheckBox] = useState(topCheckBox);
+  const [donationSiteName, setDonationSiteName] = useState('');
+  const navigate = useNavigate();
+  const { backend } = useBackend();
+
+  useEffect(() => {
+    setIndividualCheckBox(topCheckBox);
+  }, [topCheckBox]);
+
+  useEffect(() => {
+    const getDonationSiteName = async () => {
+      try {
+        const businessResponse = await backend.get(`business/${donation_site.business_id}`);
+        setDonationSiteName(businessResponse.data[0].name);
+      } catch (error) {
+        console.log(error)
+      }
+    }
+
+    getDonationSiteName();
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'backend' and 'donation_site.business_id'. Either include them or remove the dependency array.